src/main/java/info/tehnut/soulshards/core/RegistrarSoulShards.java

```diff
@@ -9,9 +9,9 @@
 import net.minecraft.block.Block;
 import net.minecraft.block.entity.BlockEntityType;
 import net.minecraft.enchantment.Enchantment;
+import net.minecraft.item.BlockItem;
```

Comment:
Unnecessary class modification

Reply:
Not sure what you mean here; the old BlockItem location was moved.

Reply:
don't mind me I'm fucking blind